app/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,358 @@
+  <string name="msg_project_not_initialized">لم تتم تهيئة المشروع</string> 
+  <string name="msg_running_task">المهمة قيد التشغيل: \ '٪ 1 $ s \'</string> 
+  <string name="msg_replacement">إستبدال</string> 
+  <string name="title_sync_project">مشروع المزامنة</string> 
+  <string name="wizard_error_name_empty">لا يمكن أن يكون الاسم فارغًا.</string> 
+  <string name="wizard_error_name_illegal">لا يمكن أن يحتوي الاسم على أحرف غير قانونية</string> 
+  <string name="wizard_save_location">حفظ الموقع</string> 
+  <string name="wizard_language">لغة المشروع</string> 
+  <string name="wizard_loading">تحميل القوالب ...</string> 
+  <string name="wizard_toolchain">معيار C ++</string> 
+  <string name="next">التالي</string> 
+  <string name="exit">مخرج</string> 
+  <string name="idepref_title_tpFix">إصلاح مؤشرات علامة JDK 11</string> 
+  <string name="idepref_msg_tpFix">اضبط LD_PRELOAD على libhook.so لإصلاح مشكلة مؤشرات العلامات لـ JDK 11 في Android 11 [DEPRECATED - تفضل باستخدام JDK 17].</string> 
+  <string name="idepref_title_customGradleInstallation">تركيب Gradle المخصص</string> 
+  <string name="idepref_msg_customGradleInstallation">حدد تثبيت Gradle مخصصًا لاستخدامه في الإنشاء. يؤدي هذا إلى تجاوز الإصدار المحدد في gradle-wrapper.properties.</string> 
+  <string name="msg_gradle_installation_path">مسار التثبيت Gradle</string> 
+  <string name="msg_gradle_installation_input_help">اتركه فارغًا لاستخدام غلاف Gradle.</string> 
+  <string name="msg_require_install_jdk_and_android_sdk">تحتاج إلى تثبيت JDK و Android SDK حتى يعمل IDE. للتثبيت ، انقر فوق \ 'نعم \' واتبع التعليمات <! [CDATA [<a href = "https://github.com/itsaky/androidide-build-tools#installing-in-androidide"> هنا </a> ]]></string> 
+ 
+  <string name="build_status_idle">لا يبني قيد التقدم</string> 
+  <string name="build_status_in_progress">البناء قيد التقدم ...</string> 
+  <string name="build_status_failed">فشل البناء. انظر بناء الإخراج لمزيد من التفاصيل.</string> 
+  <string name="build_status_sucess">بناء ناجح</string> 
+  <string name="msg_output_text_extraction_failed">فشل الحصول على نص من المحرر</string> 
+  <string name="template_def_app_name">طلبي</string> 
+  <string name="err_invalid_data_by_intent">نتج عن النية بيانات غير صالحة</string> 
+  <string name="msg_select_from_primary_storage">حدد دليلاً من التخزين الأساسي</string> 
+  <string name="err_authority_not_allowed">صلاحية الوصول ل %s غير مسموحة</string> 
+  <string name="idepref_useIcu_title">استخدم ICU lib</string> 
+  <string name="idepref_useIcu_summary">استخدم مكتبة ICU لاستخدام حواف الكلمة الاسترجاعية للنقر المزدوج واختيار الكلمات مع الضغط لفترة طويلة.</string> 
+ 
+  <!--The text sorrounded with '@@' is spanned as clickable.--> 
+  <!--There must be exactly two '@@' sorrounding the text that should be clickable.--> 
+  <string name="msg_swipe_for_files">اسحب لليسار لـfiles.</string> 
+  <string name="msg_swipe_for_output">اسحب لأعلى لـbuild output.</string> 
```

Comment:
The strings here are missing the `@@` symbols.

Reply:
I have edited them, are they changed to you or still the same ?